script.js

```diff
@@ -1,4 +1,52 @@
-var main = function (input) {
-  var myOutputValue = 'hello world';
+var main = function (userInput) {
+  // Generate a random dice number
+  var randomDiceNumber = rolldice();
+
+  //Output value is either you lose or you win
+  var myOutputValue;
+
+  // If userInput is scissors and it matches 1, update output.
+  if (userInput == "scissors") {
+    if (randomDiceNumber == 1) {
+      myOutputValue = "you draw";
```

Comment:
Another possible way if you find it better would be to list all of the scenarios that the user wins, loses or draws and put the conditions that lead to the same outcome in one if statement, separated with the 'or' operator  || between the conditions. For example,

if (userInput == "paper" && randomDiceNumber == 3) || (userInput == "stone" && randomDiceNumber == 1) || (userInput == "scissors" && randomDiceNumber == 2) {
//win outcome here
}